vue-frontend/src/components/home-new/CanopasDescription.vue

```diff
@@ -1,23 +1,23 @@
 <template>
-  <div class="container-fluid overlay">
-    <div class="container text-center">
-      <div class="description" ref="description">
+  <div class="tw-relative tw-p-0 tw-m-0 tw-py-[60px] tw-px-0">
```

Comment:
Shouldn't we have to use the `container` class of tailwind here? It will remove all other classes from here.

Reply:
I have already used tailwind class.

Reply:
tailwind has a default container class also the same as bootstrap. You can try with `tw-container`